bookshelf/tests/CloudsqlModelTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Cloud\Samples\Bookshelf;
+require_once __DIR__.'/../vendor/autoload.php';
+
+class CloudsqlModelTest extends \PHPUnit_Framework_TestCase
+{
+    use DataModelTest;
```

Comment:
I'm a little confused why this is a trait and not just part of this test. It doesn't seem like code that we'd be able to reuse.

Have you made it into a trait in hopes others will use it to test their own data models?